model/internal/pdata/generated_metrics.go

```diff
@@ -1428,14 +1428,26 @@ func (ms HistogramDataPoint) SetCount(v uint64) {
 	(*ms.orig).Count = v
 }
 
+// Sum_Type returns the type of the sum_ for this HistogramDataPoint.
+// Calling this function on zero-initialized HistogramDataPoint will cause a panic.
+func (ms HistogramDataPoint) Sum_Type() OptionalType {
```

Comment:
I think we should stick with previous public API that has a 'HasSum() bool'

Reply:
Fixed, please take a look.